### PR TITLE
feat(chat): make prompt content field required (Issue #78)

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

make prompt content field required

Issues:

- Issue #78

**UI changes**

image

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-940.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by denys-kolomiitsev — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-940.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-940.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-940.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-940.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Promptbar/components/PromptModal.tsx [hunk 1]
```diff
@@ -166,6 +179,8 @@ export const PromptModal: FC<Props> = ({ isOpen, onClose, onUpdatePrompt }) => {
     setName(selectedPrompt?.name || '');
   }, [selectedPrompt?.name]);
 
+  const saveDisabled = !name.trim() || !content.trim();
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Promptbar/components/PromptModal.tsx]:
```suggestion
  const saveDisabled = !prepareEntityName(name) || !content.trim();
```

Maybe we should think about forbidden symbols also?

**Review by Alexander-Kezik — APPROVED** [review]

**Review by denys-kolomiitsev — APPROVED** [review]

**Pull request merged**